LayoutTests/http/tests/notifications/notification.html

```diff
@@ -56,6 +64,15 @@
     assert_throws_js(TypeError, function() { new Notification("test title", { dir: "foo" }) });
 }, "The NotificationOptions dictionary only accepts valid NotificationDirection values.");
 
+promise_test(async function() {
+    let notification = new Notification("test title", { data: { foo: 'bar' } });
+    let data = notification.data;
+    assert_equals(JSON.stringify(data), '{"foo":"bar"}')
+    gc();
+    await new Promise(resolve => setTimeout(resolve, 10));
```

Comment:
Not sure how long to wait after GC here. I saw a bunch of tests using 10 ms, so I copied that here.

Reply:
I would gc() again after the timeout just to be safe :)